content/post/performance-metrics-overview.md

```diff
@@ -0,0 +1,111 @@
++++
+title = "Building low-overhead metrics collection for high-performance systems"
+date = 2018-02-20T07:20:00-05:00
+draft = false
+slug = "building-low-overhead-metrics-collection-for-high-performance-systems"
+author = "jonbrwn"
+description = "Design decisions that allowed us to capture metrics in Wallaroo with a low-overhead."
+tags = [
+    "metrics",
+    "performance",
+    "wallaroo"
+]
+categories = [
+    "Metrics",
+    "Performance",
+]
++++
+
+Metrics play an integral part in providing confidence in a high-performance software system. Whether you’re dealing with a data processing framework or a web server, metrics provide insight into whether your system is performing as expected. A direct impact of capturing metrics is the performance cost to the system they're for.
```

Comment:
Maybe change

> A direct impact of capturing metrics is the performance cost to the system they're for.

to

> A direct impact of capturing metrics is the performance cost to the system being measured.

The latter sounds a bit better to me but maybe it's not.